styles/Header.module.css

```diff
@@ -1,8 +1,333 @@
+  display: none;
+}
+
+.mobileNavMenu.open {
+  left: 0;
+}
+
+.linksNav {
+  display: inline-block;
+  padding: 25px 15px;
+  text-decoration: none;
+  color: black;
+}
+
+.activeLinkNav {
+  color: #5CA2D5;
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.mobileNavSearch {
+  position: fixed;
+  top: 0;
+  width: 350px;
+  height: 50px;
+  padding: 30px;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  right: -350px;
+  top: 75px;
+  transition: all 0.3s;
+  box-shadow: 0px 15px 15px 0 rgba(0, 0, 0, 0.15);
+  display: none;
+}
+
+.mobileNavSearch.open {
+  right: 0;
+}
+
+.searchFormNav {
+  display: inline-block;
+  height: 30px;
+  width: 100%;
+}
+
+.formButtonNav {
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+  background-image: url("/magnifying-glass.svg");
+  background-position: center;
+  background-repeat: no-repeat;   
+  border-style: solid;
+  border-width: 1px;
+  background-size: 65%;
+}
+
+
+@media (max-width: 1245px) {
+
+    .navBar{
+      padding: 0 50px;
+    }
+    
+  }
+@media (max-width: 1140px){
+
+  .navBar{
+    padding: 0px;
+  }
+  .logo {
+    flex: 1;
+    text-align: center;
+    transition: right 0.5s ease, opacity 0.1s ease;
+  }
+  .navLinks {
+    position: fixed;
+    z-index: 99;
+    top: 70px;
+    width: 100%;
+    left: -100%;
+    height: 100%;
+    padding: 10px 50px 0 50px;
+    text-align: center;
+    background: #14181f;
+    display: inline-block;
+    transition: left 0.3s ease;
+    line-height: 40px;
+    margin: 30px 0;
+  }
+ .navLinks.active{
+    left: 0px;
+  } 
+
+  .links {
+    line-height: 40px;
+    margin: 30px 0;
+    font-size: 20px;
+  }
+
+  .mobileNavSearch {
+    position: fixed;
+    top: 0;
+    width: 350px;
+    height: 50px;
+    padding: 30px;
+    background-color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    right: -350px;
+    top: 75px;
+    transition: all 0.3s;
+    box-shadow: 0px 15px 15px 0 rgba(0, 0, 0, 0.15);
+  }
+
+  .mobileNavMenu {
+    position: fixed;
+    top: 0;
+    width: 200px;
+    height: 100vh;
+    padding: 30px;
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    left: -250px;
+    transition: all 0.3s;
+    box-shadow: 0px 5px 15px 0 rgba(0, 0, 0, 0.15);
+  }
+  
+  .searchForm {
+    position: absolute;
+    top: 80px;
+    right: 50px;
+    opacity: 0;
+    pointer-events: none;
+    display: none;
+  }
+  .menuIcon {
+    display: block;
+  }
+  .searchIcon{
+    display: block;
+  }
+  .menuIcon span{
+    display: block;
+  }
+  .menuIcon span.hide{
+    display: block;
+  }
+  .searchIcon.hide{
+    display: block;
+  }
+  .cancelIcon.show{
+    display: block;
+  }
+
+}
+
+@media (max-width: 980px){
+
+    .menuIcon {
+        margin: 0 20px;
+    }
+    .searchIcon{
+        margin: 0 20px;
+    }
+
+}
+
+@media (max-width: 400px){
+
+    .mobileNavSearch {
```

Comment:
Remove this section since we're just going to make it `width: 100%` all the time